my-app/src/components/background.tsx

```diff
@@ -0,0 +1,54 @@
+import { Container, Box, Typography, CardMedia, Grid } from "@mui/material";
+
+function Background() {

```

Comment:
I suggest that we dynamically import everything as global variables instead of local require